eastwood.clj

```diff
@@ -1,49 +0,0 @@
-(disable-warning
```

Comment:
Proposing to remove this file because the linter passes with zero warnings without it.

Eastwood, because of its approach aims to be an accurate linter that needs as few workarounds as possible.

If experiencing an issue, don't hesitate reporting it upstream and we'd be pleased to either provide better defaults, or tackle false positives as such :)